src/core/components/InstallSwitch/index.js

```diff
@@ -101,7 +101,7 @@ export class InstallSwitchBase extends React.Component {
 
   render() {
     const browsertheme = JSON.stringify(getThemeData(this.props));
-    const { handleChange, slug, status } = this.props;
+    const { handleChange, slug, status, ...props } = this.props;
```

Comment:
`props` seems a bit vague, would something like `otherProps` be a bit more representative?